Formula/bareos-fd.rb

```diff
@@ -0,0 +1,72 @@
+class BareosFd < Formula
```

Comment:
what's with the `fd`? Project page seems to indicate this is just `bareos`

Reply:
I will change the description of the formula to clarify this. The formula deploys the filedaemon to interact with other components of bareos, the client side if you will.

Reply:
It seems like we should just name the formula `bareos`

Reply:
How about `BareosClient`?

Reply:
ok